scroll-animations/META.yml

```diff
@@ -0,0 +1,4 @@
+suggested_reviewers:
+  - birtles
+  - theres-waldo
+  - manta12
```

Comment:
Mantaroh is less involved in this spec recently so I think it's not necessary to include him.

Reply:
OK, I'll remove him again.